packages/block-library/src/comment-author-name/block.json

```diff
@@ -18,10 +18,6 @@
 		},
 		"textAlign": {
 			"type": "string"
-		},
-		"fontSize": {
-			"type": "string",
-			"default": "small"
```

Comment:
Don't we need to keep the attribute in order for the default template settings to work? 🤔 After all, this PR _is_ setting the blocks' `fontSize` attributes (e.g. [here](https://github.com/WordPress/gutenberg/pull/41627/files#diff-234aa0c2640d02d769e083cdea95650c0460e21902e2f2bddc63419cf872116bR43)), isn't it?

In order to achieve the desired effect, would it be enough to only remove the `default` here?

Reply:
I've tested this locally, and it seems to do the trick 🙂

Reply:
As discussed with @c4rl0sbr4v0 in Slack, I was mistaken here -- the attribute is added implicitly thanks to `fontSize` block-supports 😅